docs/reference/commandline/load.md

```diff
@@ -23,7 +23,6 @@ Load an image from a tar archive or STDIN
 Options:
       --help           Print usage
   -i, --input string   Read from tar archive file, instead of STDIN.
-                       The tarball may be compressed with gzip, bzip, or xz
```

Comment:
I just find that the content in load.md and docker-load.1.mad:

       -i, --input string   Read from tar archive file, instead of STDIN.
     -                       The tarball may be compressed with gzip, bzip, or xz

is not consistant with load.go:

    flags.StringVarP(&opts.input, "input", "i", "", "Read from tar archive file, instead of STDIN")

perhaps should add " The tarball may be compressed with gzip, bzip, or xz" to load.go?